code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/gun/ballistic/bow
+	name = "wooden bow"
+	desc = "Some sort of primitive projectile weapon. Used to fire arrows."
+	icon_state = "bow"
+	item_state = "bow"
+	w_class = WEIGHT_CLASS_BULKY
+	weapon_weight = WEAPON_HEAVY //need both hands to fire
+	force = 5
+	mag_type = /obj/item/ammo_box/magazine/internal/bow
+	fire_sound = 'sound/weapons/bowfire.wav'
+	slot_flags = ITEM_SLOT_BACK
+	item_flags = NONE
+	casing_ejector = FALSE
+	inaccuracy_modifier = 0.33 //to counteract the innaccuracy from WEAPON_HEAVY, bows are supposed to be accurate but only able to be fired with both hands
+	pin = null
+	no_pin_required = TRUE
+	trigger_guard = TRIGGER_GUARD_NONE //so ashwalkers can use it
+
+/obj/item/gun/ballistic/bow/shoot_with_empty_chamber()
+	return
+
+/obj/item/gun/ballistic/bow/chamber_round()
+	chambered = magazine.get_round(1)
+
+/obj/item/gun/ballistic/bow/afterattack()
+	. = ..()
+	if (chambered)
+		chambered = null
+		magazine.get_round(0)
```

Comment:
is this supposed to be TRUE/FALSE or 0/1?

Reply:
technically it could be FALSE since the 0 is for a variable called "keep" which refers to whether it keeps the round in the magazine but everywhere else its used is with 0/1. this could also technically be magazine.stored_ammo[1] = null I think.